index.html

```diff
@@ -64,6 +64,24 @@ <h1 class="title">Spider Man</h1>
 
     </section>
 
+        <section class="animate__animated animate__backInLeft top-gun">
```

Comment:
@IvanTurchyn all existing cards have same animate_backInLeft kind of animation - so maybe you will be able to change that to something else? 

To do that you need to go to https://animate.style website and pick different animate class than you have right now,ok? :)